aten/src/ATen/mps/MPSPinnedMemory.h

```diff
@@ -0,0 +1,9 @@
+#pragma once
+
+#include <ATen/core/Tensor.h>
+
+namespace at::mps {
+
+bool _is_pinned_ptr(const void* data);
```

Comment:
Do we need a new file for this, can this be placed next to getIMPSAllocator() ?

Reply:
It's really not necessary. Have removed this file and place it next to getIMPSAllocator() in `aten/src/ATen/mps/MPSAllocatorInterface.h`.